packages/maskbook/src/extension/background-script/CloudService.ts

```diff
@@ -0,0 +1,40 @@
+import { getStorage, setStorage } from './StorageService'
```

Comment:
You can register service under `PluginServices.Wallet`. Take a look at `packages/maskbook/src/plugins/Wallet/services/index.ts`.